src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1662,6 +1658,13 @@ components:
     # Common Response Types
     #
 
+    Epoch:
+      nullable: true
```

Comment:
Ah, damn it... this is invalid. Or rather... this is invalid name if it is meant to be nullable type. Right now we can't use this type where epoch is a mandatory property. Maybe let's just copy this definition to those two types.